subprojects/packagefiles/liburing/examples/meson.build

```diff
@@ -0,0 +1,21 @@
+executable('io_uring-cp',
+           'io_uring-cp.c',
+           include_directories : inc,
+           link_with : liburing)
```

Comment:
dependencies: uring

Since the declare_dependency combines both the include directory and the link_with into one convenient virtual interface.

Reply:
Changed. Thanks!